data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd

```diff
@@ -137,7 +133,7 @@ lifestage_encoding <- function(dat) {
                                  lifestage %in% c("fry", "button-up fry", "fry (button-up)") ~ "fry",
                                  lifestage == "smolt" ~ "smolt",
                                  lifestage == "adult" ~ "adult",
-                                 lifestage == "yearling" ~ "yearling"))
+                                 lifestage %in% c("yearling", "older juvenile") ~ "yearling"))
```

Comment:
Could older Juvenile also be a smolt/silvery parr? 
Did you look at the fork length, timing, and total amount that are categorized this way?

Reply:
Only Knights has older juvenile. They used this in their spreadsheets, not CAMP. We are only using spreadsheets pre2006 and there are no older juveniles during this time. I added a note to explain this.